kiali/README.md

```diff
@@ -65,18 +81,30 @@ If you use [Jaeger](https://www.jaegertracing.io/) for distributed tracing, Kial
 
 For integration instructions, read [Kiali: Jaeger configuration](https://kiali.io/docs/configuration/p8s-jaeger-grafana/jaeger/).
 
+To integrate with the Kyma tracing component run:
```

Comment:
```suggestion
To connect with the Kyma Tracing component, run:
```
Kyma components are [capitalized](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalization).